embedded/EscapeROOM/radio_morse_panel/source/button.c

```diff
@@ -0,0 +1,38 @@
+#include <stdint.h>
+#include "globals.h"
+#include "button.h"
+#include "misc.h"
+
+CyclicBuffer pressedButtons;
+
+void updateButtonState(void) {
```

Comment:
You moved the function definition, but forgot to move declaration from misc.h to button.h

Reply:
Correcting in next commits.